src/mimemail.erl

```diff
@@ -1070,11 +1071,18 @@ encode_extra_utf_bytes(Bytes, [C|T], AccOut) when C >= 128 andalso C =< 191 ->
 %% be located in "foo.bar._domainkey.example.com" (see RFC-6376 #3.6.2.1).
 %% `t' - signature timestamp: 'now' or UTC {Date, Time}
 %% `x' - signatue expiration time: UTC {Date, Time}
+%% `a` - key algorithm (default: `rsa-sha256`):
```

Comment:
minor: I think it's not really correct to call this "key algorithm". RFC calls it "signing algorithm"